java/src/test/java/org/certificatetransparency/ctlog/LogSignatureVerifierTest.java

```diff
@@ -47,6 +20,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.certificatetransparency.ctlog.TestData.*;
```

Comment:
Automatically done by intellij, i think i have gotten it to conform with the google style now.